draft-ietf-rtcweb-jsep.xml

```diff
@@ -2436,8 +2442,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>The "a=rtcp-mux" line MUST only be added if present in
             the most recent answer.</t>
 
-            <t>The "a=rtcp-mux-only" line MUST only be added if present
-            in the most recent answer.</t>
+            <t>If the RTP/RTCP multiplexing policy is "require",
```

Comment:
Is this right though? Do subsequent offers for existing m= sections contain mux-only?